controllers/activemqartemis_reconciler.go

```diff
@@ -2468,40 +2468,32 @@ func alder32Of(props []string) []byte {
 }
 
 func brokerPropertiesData(props []string) map[string]string {
```

Comment:
this is great, a nice simplification and a good fix.
only comment is I would like a test of this function too, the user of the newly minted and simpler code.

Reply:
Sure, I added 3 unit tests for the function BrokerPropertiesData